### DISCUSSION TITLE
API documentation > Stack choice

### DISCUSSION THREAD

**Discussion by adriens** [adriens/imgflip-api#24] [Ideas]
# Context

We want API documentation that : 

- Looks amazingly well
- Can be generated automatically
- Be customized

**Comment by adriens** [comment]
For now, we have identified [redoc](https://github.com/Redocly/redoc) , I've also found [slate](https://github.com/slatedocs/slate) that looks pretty cool

**Comment by adriens** [comment]
@HakumenNC , what do you think about [slate](https://github.com/ImgFlip-Lovers/imgflip-api/discussions/24) ❔

**Reply by HakumenNC** [reply]
Amazing, I'll take a look for sure 🤔

**Comment by adriens** [comment]
Also redoc as a SaaS version for whuch free plan looks very promising 😎 https://redoc.ly/pricing

**Comment by adriens** [comment]
OMG check that @HakumenNC  
![image](https://user-images.githubusercontent.com/5235127/126967263-76c2a5fd-0cf9-4815-9262-ee4881d3aa8c.png)
